core/PRP/Modifier/plPythonFileMod.h

```diff
@@ -26,10 +26,10 @@ struct HSPLASMA_EXPORT plPythonParameter
     {
         kInt = 1, kFloat, kBoolean, kString, kSceneObject, kSceneObjectList,
         kActivator, kResponder, kDynamicText, kGUIDialog, kExcludeRegion,
-        kAnimation, kAnimationName, kBehavior, kMaterial, kGUIPopUpMenu,
+        kAnimation, kAnimationName, kBehavior, kImage, kGUIPopUpMenu,
```

Comment:
Instead of just renaming this, perhaps we should add an alias to avoid breaking backwards compatibility?

Reply:
Do we have an existing mechanism for aliasing them?